### DISCUSSION TITLE
Pouvoir acheter des places supplémentaires dans une guilde

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#598] [Nouvelles idées]
Pouvoir acheter des places de membres dans une Guilde avec de l'argent
Il est probable qu'une augmentation du nombre de place engendre des pénalités sur le cout de l'experience de guilde.

**Comment by AlexisLhotte** [comment]
Je trouverai cela sympa, laisser la capacité à 6 place maximum au début et pouvoir acheter des places avec les gemmes par exemple.
